backend/src/main/java/de/bund/digitalservice/ris/caselaw/adapter/NormAbbreviationService.java

```diff
@@ -16,30 +16,18 @@ public NormAbbreviationService(NormAbbreviationRepository repository) {
     this.repository = repository;
   }
 
-  public Mono<NormAbbreviation> getNormAbbreviationById(UUID uuid) {
+  public NormAbbreviation getNormAbbreviationById(UUID uuid) {
     return repository.findById(uuid);
   }
 
   public Flux<NormAbbreviation> getNormAbbreviationBySearchQuery(
       String query, Integer size, Integer page) {
-
-    Integer pageOffset = null;
-    if (page != null && size != null) {
-      pageOffset = page * size;
-    }
-
-    return repository.findBySearchQuery(query, size, pageOffset);
+    return repository.findBySearchQuery(query, size, page);
   }
 
   public Mono<List<NormAbbreviation>> getNormAbbreviationByAwesomeSearchQuery(
       String query, Integer size, Integer page) {
```

Comment:
why we don't need the page anymore? if we don't needed please also remove it from the method params.